### ISSUE TITLE
Offline mode for iseries or VE migration

### ISSUE THREAD

**Issue by fjl80** [f5devcentral/f5-journeys#47]
**Is your feature request related to a problem? Please describe.**
Being able to use the tool to migrate configurations in the offline mode not only for VELOS or rseries but also to VE or iseries

**Describe the solution you'd like**
When you work with the tool and want to migrate between old TMOS versions and new versions the offline mode doesnt work for VE or iseries devices, neither its possible to choose the TMOS release you would like to go
it would be nice to be able to work in the offline mode in that way, otherwise you need the new devices connected/installed to start working on the migration tasks

Tks

**Comment by zahajkiewicz** [comment]
@fjl80 could you give more details about the failing offline scenario?
What is the source and destination version? Normally you should be able to use the application from old version to a newer one (11.x to 14.x as an example).

**Comment by fjl80** [comment]
Hey,

When you work in offline mode you cant choose the destination version neither platform series . You can only choose Velos or rSeries (what about VEs or iseries)

***@***.***

This customer I am working with needs to go from 5250V to i5800, so I cant do it in offline mode in my lab.

As far as I have checked in my lab (I don’t have a i5800) with a VE is that if I work in online mode I can migrate the config to a VE (the same config from the 5250V), so the journeys app recognize the VE and version correctly.

Is it clear my ask?

Tks




[signature_3965122646]<https://f5.com/>
 Francisco Lahoz
 M +34 687535293





From: zahajkiewicz ***@***.***>
Reply to: f5devcentral/f5-journeys ***@***.***>
Date: Friday, 1 April 2022 at 09:31
To: f5devcentral/f5-journeys ***@***.***>
Cc: Francisco Lahoz ***@***.***>, Mention ***@***.***>
Subject: Re: [f5devcentral/f5-journeys] Offline mode for iseries or VE migration (Issue #47)


EXTERNAL MAIL: ***@***.***

@fjl80<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Ffjl80&data=04%7C01%7Cf.lahoz%40f5.com%7C2fc1335f24054631cf0608da13b1af06%7Cdd3dfd2f6a3b40d19be0bf8327d81c50%7C0%7C0%7C637843951137930135%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=pEB2qF0N3h2UB0p6Rj3LziOA3Sek%2Bt24kLkfLX3z%2BJM%3D&reserved=0> could you give more details about the failing offline scenario?
What is the source and destination version? Normally you should be able to use the application from old version to a newer one (11.x to 14.x as an example).

**Comment by zahajkiewicz** [comment]
Yes, that is true. Currently Journeys App does not support offline migrations other than to VELOS tenants.
There is a plan to add support for offline migrations to a different versions/platforms.

**Comment by rjouhann** [comment]
Filed internal Jira SOLUTIONS-5766.

**Comment by jklein-f5** [comment]
Just adding my voice to supporting Offline mode for BIG-IP Classic.  As is, Journeys is still an excellent tool!
